src/test/java/lotto/LottoAnalyzerTest.java

```diff
@@ -0,0 +1,33 @@
+package lotto;
+
+import lotto.model.Lotto;
+import lotto.service.LottoAnalyzer;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+public class LottoAnalyzerTest {
+
+    @Test
+    public void 당첨통계_계산() {
+        List<Lotto> lottos = new ArrayList<>();
+
+        lottos.add(new Lotto(new ArrayList<>(List.of(2, 5, 9, 10, 23, 31))));
```

Comment:
생성자에 직접 `List.of()`로 생성한 리스트를 전달해도 되지 않을까요?
```suggestion
        lottos.add(new Lotto(List.of(2, 5, 9, 10, 23, 31)));
```